spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/controller/SkipperErrorAttributes.java

```diff
@@ -18,6 +18,7 @@
 import java.util.Map;
 
 import org.springframework.boot.web.servlet.error.DefaultErrorAttributes;
+import org.springframework.boot.web.servlet.error.ErrorAttributes;
```

Comment:
Import probably not used

Reply:
This one I explicitly imported for the Javadoc comment section. Also, the build would have failed if this isn't used ;)

Reply:
Ah, javadoc error, not build error.